Dropdown/.eslintskipwords.js

```diff
@@ -0,0 +1,62 @@
+module.exports = [
```

Comment:
I replicated those files because usually developers checkout the single component's folder, instead of the parent. That way, they will find the needed eslint files and we avoid the risk to have custom eslint files. But of course we can discuss how to optimize it.